src/local/butler/scripts/migration/monorail_to_issuetracker_migrator.py

```diff
@@ -20,7 +30,9 @@
 
 from clusterfuzz._internal.datastore import data_types
 
-DEFAULT_BATCH_SIZE = 500
+# Values more than 100 resulted in this error:
+# 400 Request payload size exceeds the limit: 11534336 bytes.
+DEFAULT_BATCH_SIZE = 100
```

Comment:
Can you catch the error and handle it (or open up a debugger)

Reply:
IIUC, this comment is similar to what Titouan suggested yesterday: "Can the script retry batches halving them, say, until it works? "
Probably can, it will make the script more complicated, and is likely not needed since 100 has been verified to work in an acceptable amount of time.

Reply:
I think opening up a debugger is cost-free though.

Reply:
```
import pdb
pdb.set_trace()
```

Reply:
Right, but normally local scripts are hacked to add debugging stmts when needed instead of checking them in. Lint warnings will also have to be bypassed for that.
Done. I can remove if you change your mind.